submissions/webdevtoday/oop_exercise/index.js

```diff
@@ -0,0 +1,76 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Mammal {
+   legs = 4;
+
+   constructor(name, gender, friends) {
+      this.name = name;
+      this.gender = gender;
+      this.friends = friends;
+   }
+}
+
+class Human extends Mammal {
+   legs = 2;
+   hands = 2;
+   species = 'human';
+
+   constructor(name, gender, saying, friends) {
+      super(name, gender, friends);
+      this.saying = saying;
+   }
+}
+
+class Animal extends Mammal {
+   constructor(name, gender, saying, species, friends) {
+      super(name, gender, friends);
+      this.saying = saying;
+      this.species = species;
```

Comment:
Looks like `species` is a shared property. Thus should be handled by a super class.

Reply:
super(name, gender, friends, species);  ?